src/main/java/com/amazonaws/secretsmanager/caching/SecretCache.java

```diff
@@ -161,4 +163,11 @@ public void close() {
         this.cache.clear();
     }
 
+    private static SecretsManagerAsyncClient createDefaultClient() {
+        ClientOverrideConfiguration config = ClientOverrideConfiguration.builder()
```

Comment:
What options do we have for updating the UserAgent for all requests instead of only the request originating from a client using this logic?  I assume the previous functionality of updating the user agent on the requests is no longer support in v2 of the SDK.  If that is the case, we may need to request for that functionality to be added back in.

Reply:
I'll revisit this.